docs/configuration/config-file-reference.md

```diff
@@ -2822,6 +2827,14 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -frontend.max-cache-freshness
 [max_cache_freshness: <duration> | default = 1m]
 
+# Maximum number of queriers that can handle requests for single user. If set to
+# 0, or value less than number of available queriers, all queriers will handle
```

Comment:
Was this meant to say "value more than number of available queriers"?

Reply:
Well spotted, fixed.